{{cookiecutter.repo_name}}/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2016, Qiita Spots
```

Comment:
Shouldn't this be a license according to the choosing of the authors? Same thing with the author name i.e. Qiita Spots should be something else, right?